@here/harp-datasource-protocol/lib/operators/InterpolationOperators.ts

```diff
@@ -4,12 +4,24 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { CallExpr, Expr, NumberLiteralExpr, Value } from "../Expr";
+import { CallExpr, Expr, ExprScope, NumberLiteralExpr, Value } from "../Expr";
 import { ExprEvaluatorContext, OperatorDescriptorMap } from "../ExprEvaluator";
 import { createInterpolatedProperty } from "../InterpolatedProperty";
 import { InterpolatedPropertyDefinition } from "../InterpolatedPropertyDefs";
 
 const operators = {
+    zoom: {
+        call: (context: ExprEvaluatorContext, args: Expr[]): Value => {
+            if (context.scope === ExprScope.Condition) {
+                // A call to 'zoom' in a filter condition expression should
+                // return the storage level of the current tile.
+                return context.env.lookup("$level")!;
```

Comment:
Why do we need to add zoom if we only support this case? It is also quite misleading b/c zoom(and level) contain the storage level offset of the data source

Reply:
This is the correct semantics. In 'filter-decoding time conditions' the operator 'zoom' must return the storage level. so user can write things like:
`when: ["step", ["zoom"], false, 14, true]`